aws/policy/paas.yaml

```diff
@@ -24,6 +29,11 @@ Statement:
   - Sid: AllowResourceRestrictedActionsWhichIncurNoFees
     Effect: Allow
     Action:
+      - cloudfront:DeleteDistribution
+      - cloudfront:UpdateDistribution
+      - cloudfront:TagResource
+      - cloudfront:UntagResource
+      - cloudfront:DeleteStreamingDistribution
```

Comment:
Small nit, but could you make sure that all the added rules are ordered alphabetically within the Sid? This hasn't been done consistently, but it can get really difficult to maintain if we don't do this.

Reply:
done